lesson21/main.js

```diff
@@ -118,24 +121,100 @@ const formingTableHeaderNameWithKey = (key) => {
 
 const createTableData = (usersData) => {
   const fragment = document.createDocumentFragment();
-  usersData.forEach((userData) => {
-    const trOfTdata = document.createElement("tr");
+  usersData.forEach((userData, index) => {
+    const trOfTdata = createElementWithClassName("tr", "users-table__tr-td");
+    const usersIndex = ++index;
     /**
      *Extract value from each of usersData
      * @param {Object} userData the object of each users data
      * @param {String,Number} val value of each users data
      */
 
-    Object.values(userData).forEach((val) => {
+    Object.keys(userData).forEach((key) => {
       const td = createElementWithClassName("td", "users-table__td");
-      td.textContent = val;
+      key === "id"
+        ? (td.textContent = usersIndex)
+        : (td.textContent = userData[key]);
```

Comment:
IMO
Since the user ID is already there, I thought I could just display it as it is instead of having to index it.
This is from when I made it.
<img width="757" alt="スクリーンショット 2022-01-04 19 12 36" src="https://user-images.githubusercontent.com/43788628/148314629-fa1161d0-5023-425b-b214-6ae31e6c83af.png">

Reply:
Since the specification of this lesson changed, I implemented this part using `memberId` and it shows like 1,2,3... :)
[119066d](https://github.com/yuka830/js-lessons/pull/27/commits/119066d2d73ff60fa315d57ba753e55d165918e3)